### DISCUSSION TITLE
Just started using Copilot, not showing suggestions

### DISCUSSION THREAD

**Discussion by TanmayAgrawal** [community/community#10074] [Copilot Conversations]
I was invited for using Copilot and hence started using it today (13/01/22). However, it is not working!
I tried using it on already tested examples from the internet, but it's not showing suggestions for me. The Copilot icon in the bottom right is exclaimed and hovering on it shows "**Copilot is encountering Temporary issues**".
I have tried reinstalling Copilot but nothing changed.

Also, I'm using WSL2
![Capture](https://user-images.githubusercontent.com/42508311/149317198-c59885e1-1b38-4fa8-9983-f75782621999.JPG)

**Comment by davecheney** [comment]
Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by fybedev** [reply]
Please make sure Copilot is actually installed. To do this, you can navigate to **C:\Users\user\.vscode\** to check if the Copilot extension's folder exists. You must also check its contents.

**Reply by davecheney** [reply]
@heewoonkim2020 from the screenshot above copilot is installed, the icon is visible in the bottom right hand corner.

**Comment by SerhiiVynohradov** [comment]
Updating Github Copilot helped (for NVim + Vundle I did :BundleUpdate).

**Comment by xvrx** [comment]
uninstalled github copilot, delete the folder in C:\Users\user.vscode\extension, install it again, worked for me
